docs/reference/release-notes/highlights-6.5.0.asciidoc

```diff
@@ -55,3 +55,14 @@ analysis. A new `multi_bucket_impact` property in the
 analysis influences the score. In {kib}, anomalies with medium or high 
 multi-bucket impact are depicted in the *Anomaly Explorer* and the 
 *Single Metric Viewer* with a cross symbol instead of a dot. 
+
+[float]
+=== Cross-cluster replication (beta)
+
+Cross-cluster replication enables you to replicate indices that exist in remote 
+clusters to your local cluster. You create an index in your local cluster 
+(known as a _follower index_) that follows an index (known as a _leader index_)
+in a remote cluster. You can also automatically follow indices in a 
+remote cluster that match a pattern. The individual write operations that occur 
+on the leader indices are then replayed on the follower indices. For more 
```

Comment:
I think we should add a sentence here, before the last sentence:

This functionality is useful for replicating data to second cluster for disaster recovery purposes, and for geo-proximity so that reads can be served locally.

I think that we should also mention here this is beta functionality and that we are excited to receive feedback. I am not sure though if this is appropriate for release notes.